tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs

```diff
@@ -39,6 +39,7 @@ public static IEnumerable<object[]> GetImageData()
         /// <summary>
         /// Verifies the SDK images contain the expected targeting packs.
         /// </summary>
+        [SkippableTheory("4.6.2", "4.7", "4.7.1", "4.7.2")]
```

Comment:
Was this absence of this causing an issue before?  I want to make sure I understand correctly.  It would be nice if there was a way to put this at the class level.

Reply:
Yeah, the test wasn't being executed before because there was no `[Theory]` attribute at all.